CHANGES.rst

```diff
@@ -17,6 +17,9 @@ debops.apache v0.1.0 - unreleased
 Added
 ~~~~~
 
+- Added a apache__redirect_to_https to control the role's default behaviour for
+  redirecting to https. [muelli_]
```

Comment:
Right, no change is needed in https://github.com/debops/ansible-apache. Ref: https://github.com/debops/debops-keyring/pull/17